app/assets/javascripts/controllers/ems_common/arbitration_profile_form_controller.js

```diff
@@ -35,6 +35,9 @@ ManageIQ.angular.app.controller('arbitrationProfileFormController', ['$scope', '
         $scope.arbitrationProfileModel.security_group_id    = convertToString(arbitrationProfileData.security_group_id);
 
         $scope.profileOptions($scope.arbitrationProfileModel.ems_id, $scope.arbitrationProfileModel.cloud_network_id);
+
+        $scope._authentication = _.find($scope.arbitrationProfileModel.authentications, {id: $scope.arbitrationProfileModel.authentication_id})
```

Comment:
Aaah, it seems `authentications` is not an array, it's an object right?

Sorry, can you give me an actual example of the data? (Like, `console.log($scope.arbitrationProfileModel.authentications)`)